.github/workflows/ci.yml

```diff
@@ -361,10 +369,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php-version:
-          - "8.0"
-          - "8.1"
-          - "8.2"
+        php-version: ${{ fromJson(needs.supported-versions-matrix.outputs.version) }}
```

Comment:
@sebastianbergmann 

Looks like we will be running additional jobs here, for

- "7.2
- "7.3"
- "7.4"